app/assets/javascripts/initialize.js

```diff
@@ -28,11 +83,11 @@ $(document).on('turbolinks:load', function() {
     switch(e.which) {
       case 37:  //left
         optionIndex = incrementOptionIndex(optionIndex, optsLength, -1);
-        break;
```

Comment:
Also nit-picky but it seems more idiomatic to align the `break` with the block's content in JavaScript.